lib/improver/tests/blending/blend_across_adjacent_points/test_TriangularWeightedBlendAcrossAdjacentPoints.py

```diff
@@ -183,7 +183,8 @@ def test_wrong_size_coords(self):
                         standard_name="lwe_thickness_of_precipitation_amount")
         new_cube.add_dim_coord(DimCoord([0, 1, 2], "forecast_period",
                                         units="hours"), 0)
```

Comment:
Change in unittest regex matching pattern required due to change in Iris2 warning text. Full error message `Require data with shape (3,), got (2,).` could not be used due to the brackets interfering with the regex matching.